etc/DependencyInstaller.sh

```diff
@@ -389,8 +396,9 @@ EOF
 
 To install or run openroad, update your path with:
     export PATH="\$(brew --prefix bison)/bin:\$(brew --prefix flex)/bin:\$(brew --prefix tcl-tk)/bin:\$PATH"
+    export CMAKE_PREFIX_PATH=\$(brew --prefix)/opt/or-tools
```

Comment:
Are you sure this is the correct path? Typically the path on M1 silicon is different from x86 silicon. I believe `$(brew --prefix)` is supposed to handle whether the package is installed in `/opt` or not?